source/_posts/2016-02-25-migrating-to-ember-cli-page-object-1-dot-0.markdown

```diff
@@ -0,0 +1,313 @@
+[here](https://github.com/san650/ember-cli-page-object/pull/91), on this
+post I will go through them and try to give some tips on how you can get
+out of trouble fast.
+
+## Change `build` calls to `create` calls.
+
+This is very simple:
+
+{% codeblock lang:js %}
+  const page = PageObject.build({
+    // ...
+  });
+{% endcodeblock %}
+
+Should be changed to:
+
+{% codeblock lang:js %}
+  const page = PageObject.create({
+    // ...
+  });
+{% endcodeblock %}
+
+## Remove `component` calls
+
+On versions `0.x` we had deprecated the `component` function, on `v1` we
+removed it completely in favor of using plain objects.
+
+{% codeblock lang:js %}
+  const page = PageObject.create({
+    // ...
+    modal: component({
+      // modal component definition
+    }),
+    // ...
+  });
+{% endcodeblock %}
+
+Should be changed to:
+
+{% codeblock lang:js %}
+  const page = PageObject.create({
+    // ...
+    modal: {
+      // modal component definition
+    },
+    // ...
+  });
+{% endcodeblock %}
+
+## Remove parentheses from helper calls
+
+On `v1` we decided to go a step further on improving the code and
+polished the tree structure we already had when defining page objects.
+The `Ceibo` project was born (you can see it over
+[here](http://github.com/san650/ceibo)) which defines a simple way of
+creating complex properties within an object. So for most cases
+properties used only to get a value, will no longer need parentheses
+when accessed.
+
+{% codeblock lang:js %}
+  const page = create({
+    scope: '#my-page',
+
+    title: text('h1'),
+    fillInName: fillable('#name')
+  });
+{% endcodeblock %}
+
+On versions `0.x` the following code was used within tests:
+
+{% codeblock lang:js %}
+  assert.equal(page.title(), 'My page title');
+  page.fillInName('Juan'); // fill #name with 'Juan'
+{% endcodeblock %}
+
+On `v1` this should be changed to:
+
+{% codeblock lang:js %}
+  assert.equal(page.title, 'My page title');
+  page.fillInName('Juan'); // Doesn't change
+{% endcodeblock %}
+
+## Element lookup
+
+On versions `0.x` defining the `scope` attribute on a page object used
+to override how the element was looked up in the DOM. Example:
+
+{% codeblock lang:js %}
+  const page = create({
+    scope: '#my-page',
+
+    title: text('h1'),
+    fillInName: fillable('#name')
+
+    modal: {
+      scope: '#my-modal',
+
+      title: PageObject.text('h3')
+    }
+  });
+{% endcodeblock %}
+
+When running tests on `0.x`:
+
+{% codeblock lang:js %}
+  page.title(); // translates to `find('#my-page h1').text()`
+  page.modal().title() //  transaltes to `find('#my-modal h3').text()`
+{% endcodeblock %}
+
+On `v1` we decided to implement scope inheritance, this means that if a
+component defines a scope and has a child component, the latter will
+inherit its parent scope.
+
+{% codeblock lang:js %}
+  page.title; // translates into find('#my-page h1').text()
+  page.modal.title //  transaltes into find('#my-page #my-modalh3').text()
```

Comment:
Also, just noticed you're sometimes using the ` symbol after "translates to" but here it's not being used